bin/config.ml

```diff
@@ -16,7 +16,7 @@ let show key =
   | None ->
       let pretty_fields = Config.pretty_fields config in
       StdLabels.List.iter pretty_fields ~f:(fun (key, value) ->
-          Logs.app (fun l -> l "%s: %s" key (show_val value)));
+          Logs.app (fun l -> l "%s: %s" key (show_val value)) );
```

Comment:
I also find it disturbing to have that extra space even for inline parenthesized expressions, is that intentional?

Reply:
The idea is to add the space to distinguish the parentheses that have been opened/closed on the same line from the ones opened/closed in other lines.
There is an exception for `fun`/`function`: the choice has been done for fun and function to only add the space before the closing parenthesis because the space after the opening parenthesis confused the users in this case (and a lot of people think it look bad).